code/datums/statistics/statcollection.dm

```diff
@@ -0,0 +1,239 @@
+/* THE GREAT BIG STATISTICS COLLECTION project
+	The objective of all this shitcode is to collect important/interesting events in a round
+	and write it to a really dumb text file, which will then be processed by an external server,
+	whichi will generate a pretty, web-viewable version (if I get my shit together)
+	by the public.
+
+	Gamemode-specific stat collection is separated off into its own files because why not
+
+
+	stat_collector is the nerve center, everything else is just there to store data until
+	round end.
+*/
+
+// Important things to store stats on that aren't located here:
+// ticker.mode, actual gamemode
+// master_mode, i.e. secret, mixed
+
+// To ensure that if output file syntax is changed, we will still be able to process
+// new and old files
+#define STAT_OUTPUT_VERSION "1.0"
+#define STAT_OUTPUT_DIR "data/statfiles/"
+
+/datum/stat_collector
+	var/enabled = 1 // unused
```

Comment:
Because there's no reason to remove it if it might get used later, which it will